lib/restAPIs/userManagement.js

```diff
@@ -0,0 +1,28 @@
+const assert = require('assert');
+
+const assertParam = require('../util/assertParam');
+const RestUtil = require('../util/RestUtil');
+
+module.exports = class RestTopic extends RestUtil {
+    _getBase(){
+        return 'users';
+    }
+    getUsers(nameBegins = '', abbreviationBegins = '', offset = 0, limit = -1){
```

Comment:
As discussed - optional parameters should be left blank and omitted from the request if empty:
```
getUsers(nameBegins, abbreviationBegins, offset = 0, limit = -1){
  const queryParameter = {
        offset: offset,
        limit: limit
  }
  
  if (nameBegins) {
    queryParameter.nameBegins = nameBegins;
  }
  
  if (abbreviationBegins) {
    queryParameter.abbreviationBegins = abbreviationBegins;
  }

  return this._makeRequest(this._getBase(), queryParameter, 'GET')
  .then(result => this.throwIfFailed(result, 200));
}
```

Reply:
Missing the "fields" param - I would add it as last in the list

Reply:
Lastly, please use indentation of 2 spaces